R/Auth.R

```diff
@@ -36,7 +36,8 @@ authorizeWebApi <- function(baseUrl, authMethod, webApiUsername = NULL, webApiPa
   # run appropriate auth. Each auth method must return a header to be added to WebAPI calls.
   authHeader <- switch(authMethod,
                        db = .authDb(baseUrl, webApiUsername, webApiPassword),
-                       ad = .authAd(baseUrl, webApiUsername, webApiPassword))
+                       ad = .authAd(baseUrl, webApiUsername, webApiPassword),
+                       windows = .authWindows(baseUrl, webApiUsername, webApiPassword))
```

Comment:
I know this is still in draft but just thought I'd ask as I looked through these changes. Should this be: `.authWindows(baseUrl, ":",":")` instead?

Reply:
That doesn't work on a OS X which is where I wrote the code. I'm not sure it will work on windows either as I'm yet to test it there.

Reply:
Ah good to know. I'll give it a try on Windows as-is to see how it goes.